File: programmers/Level0/solution/_184_0_떼기/Solution.java
package programmers.Level0.solution._184_0_떼기;

/**
 * https://school.programmers.co.kr/learn/courses/30/lessons/181847?language=java
 *
 */
class Solution {

    public static void main(String[] args) {
        String n_str = "0010";
        String expectedResult = "10";

        Solution solution = new Solution();
        String result = solution.solution(n_str);

        if (expectedResult.equals(result)) {
            System.out.println("Pass");
        } else {
            System.out.println("Fail");
        }
    }

    public String solution(String n_str) {
        return String.valueOf(Integer.parseInt(n_str));
    }
}

File: programmers/Level0/solution/_184_0_떼기/Solution.kt
package programmers.Level0.solution._184_0_떼기

fun main() {
    val n_str = "0010"
    val expectedResult = "10"

    val result = solution(n_str)

    if (expectedResult == result) {
        println("Pass")
    } else {
        println("Fail")
    }
}

fun solution(n_str: String): String {
    return n_str.toInt().toString()
}
